blogs/converter.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+const marked = require('marked');
+
+var listOfDirectoryFiles = fs.readdirSync('markdown-blogs');
+
+console.log(listOfDirectoryFiles);
+
+var listOfMarkdownFiles = [];
```

Comment:
All of this is good and works but to be more modern, you might want to look at more of the methods on an array.
`listOfDirectoryFiles` you might want to use a map to be more functional and cleaner:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

Reply:
Since the markdown files are all in a separate folder I can skip this step now.